chaoscenter/subscriber/pkg/utils/definations.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"subscriber/pkg/graphql"
+	"subscriber/pkg/k8s"
+)
+
+var gqlSubscriberServer = graphql.NewGqlServer()
+var subscriberK8s = k8s.NewKubernetes()
```

Comment:
If we are moving to a more interface/method oriented approach global variables should be avoided, these should rather be struct members.